packaging/osx/scripts/postinstall

```diff
@@ -4,6 +4,9 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
 
+# A temporary fix for the permissions issue(s)
+chmod -R 755 $2
```

Comment:
As discussed please make a comment here on what values $2 (and potentially $1) represent. May save some headache later :smile: